lib/python/picongpu/picmi/phase_space.py

```diff
@@ -0,0 +1,72 @@
+"""
+This file is part of PIConGPU.
+Copyright 2021-2024 PIConGPU contributors
+Authors: Masoud Afshari
+License: GPLv3+
+"""
+
+from ..pypicongpu.output.phase_space import PhaseSpace as PyPIConGPUPhaseSpace
+from ..pypicongpu.species.species import Species as PyPIConGPUSpecies
+
+
+from .species import Species as PICMISpecies
+
+import picmistandard
+import typeguard
+from typing import Literal
+
+
+@typeguard.typechecked
+class PhaseSpace(picmistandard.PICMI_PhaseSpace):
+    """PICMI object for Phase Space diagnostics"""
```

Comment:
The changes in the PICMI standard are not yet mainline, so we can't rely on having this as a dependency. A user unknowingly trying to use this after downloading the mainline PICMI standard would fail to do so. I'd suggest we do the following:
```suggestion
import typeguard
from typing import Literal


@typeguard.typechecked
class PhaseSpace:
    """PICMI object for Phase Space diagnostics"""
```
If we agree to do this, we'd also need to copy over the docstring from [here](https://github.com/mafshari64/picmi/blob/e99f008c14259d8f65212e73f0009abe35aeb781/PICMI_Python/phase_space.py#L9C1-L37C8). In that docstring, please add information about the expected units.

Reply:
done

Reply:
This is not yet done. There's no docstring yet.

Reply:
I forgot, I added now info to [picmi/phase_space.py. I push it soon.